app/models/constants/case_log.rb

```diff
@@ -0,0 +1,1134 @@
+module Constants::CaseLog
```

Comment:
I think this file was removed in main so it shouldn't be in this PR? 🤔

Reply:
removed

Reply:
Looks like it's still here? (This whole file)